provider/src/main/kotlin/org/gradle/kotlin/dsl/SettingsExtensions.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.kotlin.dsl
+
+import org.gradle.api.initialization.Settings
+
+/**
+ * Adds the given projects to the build.
+ *
+ * @param projectPaths the projects to add
+ * @see [Settings.include]
+ */
+fun Settings.include(vararg projectPaths: String) {
```

Comment:
⭕️  I assume we're adding this here because it would be a breaking change to do it on the Gradle side?

Reply:
Please excuse my lack of knowledge of the java language, but wouldn't changing:
```java
void include(String[] projectPaths);
```
to
```java
void include(String... projectPaths);
```
still allow you to pass an array into the `include` method? Maybe this is just a misunderstanding of Java method signatures.

Reply:
I removed these premature extensions and postponed this aspect as #543